ikarus-webapp/src/features/base/model/BaseModulePrimaries.js

```diff
@@ -5,6 +5,10 @@ BaseModulePrimary1 = function BaseModulePrimary1(){
   this.description = "An irregular camp of couple tents, a fire and two ATVs." +
    " Most basic of possible primary modules for a base.";
   this._primary = true;
+  this._removeLoot = {
+    'hlc_rifle_aks74u': 4,
+    'hlc_30Rnd_545x39_B_AK': 20
+  };
 };
```

Comment:
Turns out that the box having 4 aks-74u and 20 magazines cause them to be added as mission loot, and each mission basically gives this many weapons to each company. NO LONGER!